CHANGELOG-prerelease.md

```diff
@@ -1,3 +1,10 @@
+## Prerelease 42 ( TBD )
+
+Tag: [v1.0.0-prerelease.42](https://github.com/patternfly/patternfly-elements/releases/tag/v1.0.0-prerelease.42)
+
+- [](https://github.com/patternfly/patternfly-elements/commit/) fix: set consistent line-height for pfe-nav triggers  #790
```

Comment:
can we add the issue number too @chrisdo1  https://github.com/patternfly/patternfly-elements/issues/773

Reply:
@starryeyez024 changed that hash to refer to the original issue.